tests/cypress/tests/unit/lexeme_test.ts

```diff
@@ -0,0 +1,111 @@
+import {
+    fieldExpression,
+    stringExpression,
+    valueExpression
+} from 'dash-table/syntax-tree/lexeme/expression';
+import operand from 'dash-table/syntax-tree/lexeme/operand';
+import { ISyntaxTree } from 'core/syntax-tree/syntaxer';
+
+describe('expression', () => {
+    it('resolves field expression', () => {
+        expect(!!fieldExpression.resolve).to.equal(true);
+        expect(typeof fieldExpression.resolve).to.equal('function');
+
+        if (fieldExpression.resolve) {
+            expect(fieldExpression.resolve({ abc: 3 }, { value: '{abc}' } as ISyntaxTree)).to.equal(3);
+            expect(fieldExpression.resolve({ ['a bc']: 3 }, { value: '{a bc}' } as ISyntaxTree)).to.equal(3);
+            expect(fieldExpression.resolve({ ['{abc}']: 3 }, { value: '{\\{abc\\}}' } as ISyntaxTree)).to.equal(3);
+            expect(fieldExpression.resolve({ ['"abc"']: 3 }, { value: '{"abc"}' } as ISyntaxTree)).to.equal(3);
+
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '3' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: 'abc' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '{abc' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: 'abc}' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '}abc{' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '{{abc}}' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '"' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '`' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: `'` } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '{' } as ISyntaxTree)).to.throw(Error);
+            expect(fieldExpression.resolve.bind(undefined, undefined, { value: '}' } as ISyntaxTree)).to.throw(Error);
```

Comment:
Perhaps include a `target` here, so we know these errors aren't just from failed lookup. Also would be nice to demonstrate what happens with a real failed lookup, ie:
`fieldExpression.resolve({ abc: 3 }, { value: '{def}' } as ISyntaxTree)`

Reply:
Adding